modules.d/01fips-crypto-policies/fips-crypto-policies.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/sh
+
+type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
+
+if ! fipsmode=$(getarg fips) || [ "$fipsmode" = "0" ] || [ -z "$fipsmode" ]; then
```

Comment:
should the script proceed on anything but 1 there?

Reply:
I copied and modified this from the existing `01fips` module, so it's consistent with that. I don't believe we should introduce a difference between those two.

Reply:
can't argue with that